[2024-12-14T11:27:50.385+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-14T11:27:50.416+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ecobalyse_dag.linear_regression manual__2024-12-14T11:27:29.828784+00:00 [queued]>
[2024-12-14T11:27:50.431+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ecobalyse_dag.linear_regression manual__2024-12-14T11:27:29.828784+00:00 [queued]>
[2024-12-14T11:27:50.431+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 3
[2024-12-14T11:27:50.450+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): linear_regression> on 2024-12-14 11:27:29.828784+00:00
[2024-12-14T11:27:50.463+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=4525) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-14T11:27:50.465+0000] {standard_task_runner.py:72} INFO - Started process 4548 to run task
[2024-12-14T11:27:50.466+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ecobalyse_dag', 'linear_regression', 'manual__2024-12-14T11:27:29.828784+00:00', '--job-id', '644', '--raw', '--subdir', 'DAGS_FOLDER/data_transformations.py', '--cfg-path', '/tmp/tmpx5z1n7ae']
[2024-12-14T11:27:50.467+0000] {standard_task_runner.py:105} INFO - Job 644: Subtask linear_regression
[2024-12-14T11:27:50.532+0000] {task_command.py:467} INFO - Running <TaskInstance: ecobalyse_dag.linear_regression manual__2024-12-14T11:27:29.828784+00:00 [running]> on host a07fab1983be
[2024-12-14T11:27:50.628+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='thiernosidybah232@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ecobalyse_dag' AIRFLOW_CTX_TASK_ID='linear_regression' AIRFLOW_CTX_EXECUTION_DATE='2024-12-14T11:27:29.828784+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-14T11:27:29.828784+00:00'
[2024-12-14T11:27:50.629+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-12-14T11:27:50.630+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-12-14T11:27:50.630+0000] {logging_mixin.py:190} INFO - Current task name:linear_regression state:running start_date:2024-12-14 11:27:50.417637+00:00
[2024-12-14T11:27:50.630+0000] {logging_mixin.py:190} INFO - Dag name:ecobalyse_dag and current dag run status:running
[2024-12-14T11:27:50.630+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-14T11:27:50.631+0000] {logging_mixin.py:190} INFO - ---Linear Regression-----------------
[2024-12-14T11:27:50.714+0000] {taskinstance.py:3311} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 417, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/data_transformations.py", line 401, in Linear_Regression_Model
    data_regressor = pd.DataFrame(result)
                     ^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/core/frame.py", line 733, in __init__
    mgr = dict_to_mgr(data, index, columns, dtype=dtype, copy=copy, typ=manager)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/core/internals/construction.py", line 503, in dict_to_mgr
    return arrays_to_mgr(arrays, columns, index, dtype=dtype, typ=typ, consolidate=copy)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/core/internals/construction.py", line 114, in arrays_to_mgr
    index = _extract_index(arrays)
            ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/core/internals/construction.py", line 677, in _extract_index
    raise ValueError("All arrays must be of the same length")
ValueError: All arrays must be of the same length
[2024-12-14T11:27:50.745+0000] {logging_mixin.py:190} INFO - Task instance in failure state
[2024-12-14T11:27:50.745+0000] {logging_mixin.py:190} INFO - Task start:2024-12-14 11:27:50.417637+00:00 end:2024-12-14 11:27:50.744592+00:00 duration:0.326955
[2024-12-14T11:27:50.745+0000] {logging_mixin.py:190} INFO - Task:<Task(PythonOperator): linear_regression> dag:<DAG: ecobalyse_dag> dagrun:<DagRun ecobalyse_dag @ 2024-12-14 11:27:29.828784+00:00: manual__2024-12-14T11:27:29.828784+00:00, state:running, queued_at: 2024-12-14 11:27:29.839727+00:00. externally triggered: True>
[2024-12-14T11:27:50.745+0000] {logging_mixin.py:190} INFO - Failure caused by All arrays must be of the same length
[2024-12-14T11:27:50.746+0000] {taskinstance.py:1225} INFO - Marking task as UP_FOR_RETRY. dag_id=ecobalyse_dag, task_id=linear_regression, run_id=manual__2024-12-14T11:27:29.828784+00:00, execution_date=20241214T112729, start_date=20241214T112750, end_date=20241214T112750
[2024-12-14T11:27:50.758+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-14T11:27:50.759+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 644 for task linear_regression (All arrays must be of the same length; 4548)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3005, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3159, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3183, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 417, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/data_transformations.py", line 401, in Linear_Regression_Model
    data_regressor = pd.DataFrame(result)
                     ^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/core/frame.py", line 733, in __init__
    mgr = dict_to_mgr(data, index, columns, dtype=dtype, copy=copy, typ=manager)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/core/internals/construction.py", line 503, in dict_to_mgr
    return arrays_to_mgr(arrays, columns, index, dtype=dtype, typ=typ, consolidate=copy)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/core/internals/construction.py", line 114, in arrays_to_mgr
    index = _extract_index(arrays)
            ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/core/internals/construction.py", line 677, in _extract_index
    raise ValueError("All arrays must be of the same length")
ValueError: All arrays must be of the same length
[2024-12-14T11:27:50.800+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-12-14T11:27:50.831+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-14T11:27:50.833+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
